### DISCUSSION TITLE
nightly: appveyor download limit exceed message

### DISCUSSION THREAD

**Discussion by bachig26** [QL-Win/QuickLook#1284] [Q&A]
I just found the QuickLook and tried to go for nightly build with latest commit https://github.com/QL-Win/QuickLook/commit/592a768cd34580c052114d7e1802ee003f657c2b.

But when I tried to download the either of the packages `.msi` or `zip`, I'm getting the `Artifacts download limit exceeded message as I provided below`. 
AppVeyor is new to me and not sure of the issue here, so I'm looking for some help, as this build has only published with an hour before this discussion. Furthermore, I haven't used AppVeyor before or with my current IP.

```
{"message":"Artifacts download limit (1024 MB/day) exceeded."}
```

**Comment by rabelux** [comment]
Try again tomorrow. AppVeyor seems to have a global limit per day and not per binary/user.

**Reply by bachig26** [reply]
thank you. i'll try it tomorrow then. ✌

**Reply by bachig26** [reply]
@rabelux will there be any possibility to have an alternate storage for nightly builds in addition to AppVeyor in future?

**Reply by rabelux** [reply]
Personally I'd welcome a shift to a different CI/CD provider as the daily limit is the second downside after the expiry of artifacts after 30 days. I'd love to hear what @xupefei thinks about it.
But even if he agrees the question is whether the pain is big enough to invest the required time and effort.

**Reply by xupefei** [reply]
For me the problem is to find such a CI/CD that is free and solves the problem :(

**Reply by mooflu** [reply]
I tried a POC using github actions here: https://github.com/mooflu/QuickLook/actions/workflows/msbuild.yml 
It does a build and then adds the zip & msi artifacts to a dummy ["nightly" release](https://github.com/mooflu/QuickLook/releases). 

<edit>
removed above and add PR: https://github.com/QL-Win/QuickLook/pull/1337

**Reply by bachig26** [reply]
> I tried a POC using github actions here: https://github.com/mooflu/QuickLook/actions/workflows/msbuild.yml 
> 
> It does a build and then adds the zip & msi artifacts to a dummy ["nightly" release](https://github.com/mooflu/QuickLook/releases). 

I hope it gets merged or added to the repo.
